### PR TITLE
Documentation: Add FAQ items for reverse proxy and rpi performance

### LINKED ISSUE #2064: [FAQ Update] Machine Learning Raspberry Pi performance issues
### The bug

As requested in Discord, Im submitting this issue here in order to request to have warnings added to the FAQ for users who are installing Immich on Raspberry Pi systems.

Default immich deployment includes machine learning containers, however in my experience this slows the raspberry pi (4 in my case) down to the point the machine becomes completely unresponsive for indefinite periods of time.

Commenting out the machine learning container from docker-compose.yml and updating the .env file with IMMICH_MACHINE_LEARNING_URL=false fixed the problem for me. The pi is once again responsive and usable again even under heavy load from syncing many files.

### The OS that Immich Server is running on

Raspbian 64bit

### Version of Immich Server

v1.51.2

### Version of Immich Mobile App

1.51

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
*
```

### Your .env content

```Shell
*
```

### Reproduction steps

```bash
1.
2.
3.
...
```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by etnoy**
This adds FAQ items for disabling machine learning on rpi and setting POST sizes on a reverse proxy.

Also some minor cleanup and re-wording.

Fixes #2064

### CODE REVIEW

File: docs/docs/FAQ.md [hunk 1]
```diff
@@ -14,7 +14,11 @@ sidebar_position: 7
 
 ### How can I sync an existing directory with Immich's server?
 
-Immich doesn't have the mechanism to sync an existing directory with the server. There is however, a helper CLI tool to help you bulk upload the existing photos and videos to the server. You can find the guide to use the CLI tool [here](/docs/features/bulk-upload.md).
+Immich doesn't have two-way synchronization ([yet](https://github.com/immich-app/immich/discussions/1006)), but the [command line tool](/docs/features/bulk-upload.md) can bulk upload items from a directory to Immich.
+
+### Why doesn't Immich watch an existing photo gallery directory?
+
+The initial approach of Immich is to become a backup tool, primarily for mobile device usage. Thus, all the assets must be uploaded from the mobile client. The app was architectured to perform that job well, however contributions are welcome to add more features.
```
Comment (bo0tzz) [inline docs/docs/FAQ.md:21]: The "contributions welcome" part in this context seems to imply that a PR for folder watching is welcome, which I think isn't the case? @alextran1502 ?

Comment (etnoy) [reply docs/docs/FAQ.md:21]: That's probably not worded right, let me fix

**Review by bo0tzz — APPROVED** [review]

**Review by jrasm91 — APPROVED** [review]

File: docs/docs/FAQ.md [hunk 2]
```diff
@@ -29,14 +33,19 @@ As an example, the following modification of ```docker-compose.yml``` will set t
...
 
-The initial approach of Immich is to become a backup tool, primarily for mobile device usage. Thus, all the assets must be uploaded from the mobile client. The app was architectured to perform that job well.
+### Why is Immich slow on low-memory systems like the Raspberry Pi?
+Immich uses optional machine-learning features to enhance search results. This feature, however, can be too heavy to run on a Raspberry Pi. To disable machine learning, comment out the `immich-machine-learning` section of your docker-compose.yml and set `IMMICH_MACHINE_LEARNING_URL=false` in your .env file.
 
 ### What happens to existing files after I choose a new [Storage Template](/docs/administration/storage-template.mdx)?
 
 Template changes will only apply to new assets. To retroactively apply the template to previously uploaded assets, run the Storage Migration Job, available on the [Jobs](/docs/administration/jobs.md) page.
 
+### In the uploads folder, why are photos stored in the wrong date?
```
Comment (alextran1502) [inline docs/docs/FAQ.md:46]: Hello, can you elaborate on this? For the current implementation, this should not happen

Comment (jrasm91) [reply docs/docs/FAQ.md:46]: I think that on file upload the asset.fileCreatedAt defaults to the file created date and that's what is used in the storage template. Then after exif extraction saves the corrected date, the new storage template re-applied with the now changed date.

**Pull request merged**